kolibri/core/assets/src/views/ExamReport/__tests__/TriesOverview.spec.js

```diff
@@ -0,0 +1,68 @@
+import { render, screen } from '@testing-library/vue';
+import '@testing-library/jest-dom';
+import VueRouter from 'vue-router';
+import TriesOverview from '../TriesOverview.vue';
+import * as tryValidatorModule from '../utils';
+
+// Mock the tryValidator namespace as the same is used in the component
+// eslint-disable-next-line import/namespace
+tryValidatorModule.tryValidator = jest.fn(() => true);
+
+// Helper function to render the component with some default props
+const renderComponent = props => {
+  const commonCoreStrings = {
+    methods: {
+      coreString: x => x,
+    },
+  };
+
+  const defaultProps = {
+    pastTries: [],
+    totalQuestions: 20,
+    suggestedTime: 240,
+    ...props,
+  };
+
+  return render(TriesOverview, {
+    props: {
+      ...defaultProps,
+      ...props,
```

Comment:
Double defaulting here in addition to line 23.

Reply:
Nice catch! Fixed in the latest commit. Thanks.